include/blogPost.php

```diff
@@ -0,0 +1,48 @@
+<?php
+function getArray(){
```

Comment:
It looks like you and Kelly were working on this together. That's not a problem (it's actually awesome, I hope you keep it up) but I left some comments for her about the name of this function. It might make sense for all three of us to talk about naming functions.

The gist of what I told her is that this function name seems a bit too generic. `GetArray` doesn't say much. As you write more code, you'll end up with a bunch of functions that return different types of arrays. What you want convey with this function name is something like `GetAllBlogPosts`